tools/clang/test/CodeGenSPIRV/hull.structure.hlsl

```diff
@@ -0,0 +1,38 @@
+// Run: %dxc -T hs_6_0 -E main
+
+#include "bezier_common_hull.hlsl"
+
+// This test ensures:
+// 1- The hull shader entry point function is invoked.
+// 2- Control barrier is placed before PCF is called.
+// 3- PCF is called by invocationID of 0.
+
+// CHECK:         %main = OpFunction %void None {{%\d+}}
+// CHECK:   [[id:%\d+]] = OpLoad %uint %gl_InvocationID
+
+// CHECK:      {{%\d+}} = OpFunctionCall %BEZIER_CONTROL_POINT %src_main %param_var_ip %param_var_i %param_var_PatchID
+
```

Comment:
Need to check that we've stored the result before the barrier.

Reply:
Done